applications/nrf_desktop/README.rst

```diff
@@ -1831,6 +1831,43 @@ These are valid for events that have many listeners or sources, and are gathered
    doc/hfclk_lock.rst
    doc/event_rel_modules.rst
 
+Application configuration API
+*****************************
+
+Listed below are elements of API used by application configuration.
+
+HID reports
+===========
+
+| Header file: :file:`applications/nrf_desktop/configuration/common/hid_report_desc.h`
```

Comment:
Please give the details of c files also: For example, Source files: file:`applications/nrf_desktop/configuration/common/hid_report_desc.c`

Also, in all the below cases.

Reply:
Do we really want to do it? I personally would try to expose only the API here not the details of implementation.

Reply:
I mean to provide the Source file name. For e.g: http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-7484/nrf/applications/asset_tracker_v2/doc/util_module.html#api-documentation

![image](https://user-images.githubusercontent.com/91891495/169292608-d9d9ad79-dc59-4c07-87a4-433075debf1e.png)

Reply:
should be ok now